term/src/terminalstate/mouse.rs

```diff
@@ -56,6 +56,8 @@ impl TerminalState {
             MouseButton::Right => 2,
             MouseButton::WheelUp(_) => 64,
             MouseButton::WheelDown(_) => 65,
+            MouseButton::WheelLeft(_) => 66, //TODO: check
```

Comment:
Confirmed; 66 is Button6 which is the left wheel, and 67 is Button7, the right wheel

Reply:
Awesome, thanks! Out of curiosity, where do you find this information?

Reply:
https://www.invisible-island.net/xterm/ctlseqs/ctlseqs.html#h3-Other-buttons 
plus other code here in wezterm's repo at:

https://github.com/wez/wezterm/blob/dbed5017f828e8719ad871a031acd4c09c0c98d0/termwiz/src/escape/csi.rs#L605-L606

Reply:
It should be gone now